Python/ceval.c

```diff
@@ -1318,7 +1318,7 @@ missing_arguments(PyThreadState *tstate, PyCodeObject *co,
     Py_ssize_t i, j = 0;
     Py_ssize_t start, end;
     int positional = (defcount != -1);
-    const char *kind = positional ? "positional" : "keyword-only";
+    const char *kind = positional ? "positional" : "keyword";
```

Comment:
Why not keeping keyword-**only** here, it's correct, no?

```
>>> def f(*, x): pass
... 
>>> f()
Traceback (most recent call last):
  File "<python-input-1>", line 1, in <module>
    f()
    ~^^
TypeError: f() missing 1 required keyword-only argument: 'x'
```

Reply:
"Keyword-only" is a qualifier of a parameter. Arguments can be "positional" or "keyword".